Source/JavaScriptCore/runtime/IntlDateTimeFormat.cpp

```diff
@@ -739,15 +723,15 @@ void IntlDateTimeFormat::initializeDateTimeFormat(JSGlobalObject* globalObject,
     String tz;
     String timeZoneForICU;
     if (!tzValue.isUndefined()) {
-        String originalTz = tzValue.toWTFString(globalObject);
+        const String originalTz = tzValue.toWTFString(globalObject);
```

Comment:
Why this change?

Reply:
That’s true of thousands of other local variables in functions across WebKit, and as a matter of coding style we don’t use `const` for those. We could start but I don’t see why we’d start here.

Reply:
Although that makes sense, it's not a typical WebKit coding style to use const locals.

Reply:
Got it. I'll remove this `const`.